src/Chapter2.hs

```diff
@@ -728,7 +749,7 @@ value of the element itself
 🕯 HINT: Use combination of 'map' and 'replicate'
 -}
 smartReplicate :: [Int] -> [Int]
-smartReplicate l = error "smartReplicate: Not implemented!"
+smartReplicate = concatMap(\l -> replicate l l)
```

Comment:
Just a minor stylistic comment on adding spaces between function and its arguments :slightly_smiling_face: 
```suggestion
smartReplicate = concatMap (\l -> replicate l l)
```